src/test/java/com/github/tomakehurst/wiremock/ProxyAcceptanceTest.java

```diff
@@ -407,7 +406,7 @@ public void doesNotDuplicateCookieHeaders() {
     testClient.get("/duplicate/cookies", withHeader("Cookie", "session=1234"));
 
     LoggedRequest lastRequest =
-        getLast(target.find(getRequestedFor(urlEqualTo("/duplicate/cookies"))));
+        getFirst(target.find(getRequestedFor(urlEqualTo("/duplicate/cookies"))), null);
```

Comment:
I think this will be a breaking change for a lot of folks. How about we modify AbstractRequestJournal.getRequestsMatching(...) to reverse the results so that they'll come back in the same order as previously.

Reply:
But we will return to what we left...
Breaking changes it's normal when update version IMHO...
And as a user, I'm waiting in "history" for more recent entries to be at the top...

Reply:
I mean reverse the result of the find query rather than the whole list, which would have a pretty trivial impact by comparison and only affect finding vs. fetching the whole journal.

Agree it's in principle acceptable to create breaking changes, but this one is avoidable and my instinct is that it'll annoy people.